lib/docker.sh

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env bash
+
+# PROMPT COLOURS
+readonly RESET='\033[0;0m'
+readonly BLACK='\033[0;30m'
+readonly RED='\033[0;31m'
+readonly GREEN='\033[0;32m'
+readonly YELLOW='\033[0;33m'
+readonly BLUE='\033[0;34m'
+readonly PURPLE='\033[0;35m'
+readonly CYAN='\033[0;36m'
+readonly WHITE='\033[0;37m'
+
+readonly DOCKER_MINIMAL_VERSION=18.04.0
+readonly DOCKER_COMPOSE_MINIMAL_VERSION=1.24.0
+
+function check_requirements() {
+    check_docker
+    check_docker_compose
+    check_jq
+}
+
+function check_version () {
+    local version=$1
+    local require_version=$2
+    local package=$3
+
+    dpkg --compare-versions ${version} 'ge' ${require_version} \
+        || (echo -e "${RED}Requirement: need '${package}:${require_version}', you are '${package}:${version}'." && exit 1)
+}
+
+function check_docker() {
+    if [[ "$(which docker)" == "" ]]; then
+        echo -e "${RED}Requirement: need 'docker:${DOCKER_MINIMAL_VERSION}' see https://docs.docker.com/install/linux/docker-ce/ubuntu ."
+        exit 1
+    fi
+
+    check_version $(docker -v | sed 's/,//'| awk '{print $3}') ${DOCKER_MINIMAL_VERSION} 'docker'
+}
+
+function check_docker_compose() {
+    if [[ "$(which docker-compose)" == "" ]]; then
+        echo -e "${RED}Requirement: need 'docker-compose:${DOCKER_COMPOSE_MINIMAL_VERSION}' see https://docs.docker.com/compose/install ."
```

Comment:
ça foire le lien si on met le `.` collé au lien ?
```suggestion
        echo -e "${RED}Requirement: need 'docker-compose:${DOCKER_COMPOSE_MINIMAL_VERSION}' see https://docs.docker.com/compose/install."
```

Reply:
le lien n'est plus bon si je colle le point à l'url